### ISSUE TITLE
Font referenced in scss file not loading

### ISSUE THREAD

**Issue by tommers-walker** [gridsome/gridsome#949]
I've included some fonts in the assets directory as per the documentation. I can reference the fonts from within a component but if I try to reference them in a scss file, Gridsome throws an error:

"This relative module was not found: * ../assets/fonts/Gellix-Regular.otf in ./node_modules/css-loader/dist/cjs.js??ref--4-oneOf-1-1!./node_modules/postcss-loader/src??ref--4-oneOf-1-2!./node_modules/sass-loader/dist/cjs.js!./node_modules/style-resources-loader/lib??ref--4-oneOf-1-4!./src/assets/styles/main.scss"

![Screenshot 2020-01-20 at 16 47 23](https://user-images.githubusercontent.com/25368754/72744049-bd9bc880-3ba4-11ea-98d3-ad7e17e49afe.png)
![Screenshot 2020-01-20 at 16 47 43](https://user-images.githubusercontent.com/25368754/72744052-be345f00-3ba4-11ea-9026-9af3b4e5bba8.png)

I've had to put them in the static directory for now

**Comment by hjvedvik** [comment]
It seems to me that the path should be `../fonts/Gellix-Regular.otf` instead. Can you try that and see if it helps?

**Comment by syropian** [comment]
FWIW I'm running into the same issue, and my file paths are definitely correct.

**Comment by Revii** [comment]
Same here, they are in assets folder, but not load on page view.

**Comment by Revii** [comment]
OK, it works if you remove ```system-ui``` from font-family.

**Comment by a-kriya** [comment]
@tommers-walker @syropian Were you able to resolve the issue?

**Comment by syropian** [comment]
@a-kriya Sort of, but not really — ended up adding the fonts to the static folder instead.

**Comment by ChungMasterFlex** [comment]
@a-kriya  Can you try this: '~@/assets/fonts/Gellix-Regular.otf' as your path instead? 

@syropian <-- it sounds like you able to handle routing issues -- but the font is not being picked up - is that correct?

**Comment by a-kriya** [comment]
I was just curating issues; haven't encountered this as I load fonts from a .css in my project. But I think what you suggested, @ChungMasterFlex, should work for @syropian. I use the `~@` alias in some SCSS blocks in components and in `url()`.

**Comment by syropian** [comment]
Yeah, it seems to be actually working ok for me, just using relative paths now. I'm not sure what was happening before 🤔

**Comment by ChungMasterFlex** [comment]
It does seem to sometime take a little while for the changes to “register”
- I had to do a little voodoo magic / reset router local / recreate & save
pages / also I had to make sure the css ID name was not matching the
component name in the export! This last gotcha took me a little while to
figure out.
